src/20_Components/amp-ad.html

```diff
@@ -53,6 +57,19 @@
     </div>
   </amp-ad>
 
+  <!-- #### Sticky Ads -->
+  <!--
+    `amp-sticky-ad` adds supports for ad units that always take a fixed place in the viewport.
```

Comment:
These units are often called "adhesion" units, so I'd add that word for folks who are more familiar with it